internal/cmd/feedback/command.go

```diff
@@ -0,0 +1,74 @@
+package feedback
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	cliv1 "github.com/confluentinc/ccloud-sdk-go-v2/cli/v1"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	pversion "github.com/confluentinc/cli/internal/pkg/version"
+)
+
+type command struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+func New(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "feedback",
+		Short:       fmt.Sprintf("Submit feedback for the %s.", pversion.FullCLIName),
+		Args:        cobra.NoArgs,
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLogin},
+	}
+
+	c := &command{
+		AuthenticatedCLICommand: pcmd.NewAuthenticatedCLICommand(cmd, prerunner),
+	}
+	cmd.RunE = c.feedback
+
+	return cmd
+}
+
+func (c *command) feedback(_ *cobra.Command, _ []string) error {
+	feedback, err := getFeedback(form.NewPrompt(os.Stdin))
+	if err != nil {
+		return err
+	}
+	if len(feedback) > 0 {
+		if err := c.sendFeedback(feedback); err != nil {
+			return err
+		}
+		fmt.Println("Thanks for your feedback.")
+	}
+	return nil
+}
+
+func getFeedback(prompt form.Prompt) (string, error) {
+	f := form.New(form.Field{
+		ID:        "proceed",
+		Prompt:    "Please confirm that your feedback does not contain any sensitive information",
```

Comment:
It seems weird to prompt the user with this before they've actually written the feedback. I think there are two better ways to do this:
1. Print a message reminding them to not include sensitive information without a prompt, then collect their feedback
2. Collect their feedback, then show this prompt (I think this is what the security team wants)

Reply:
I think we should do option 2